src/NUnitEngine/nunit.engine/Internal/Logging/Logger.cs

```diff
@@ -165,9 +159,14 @@ private void WriteLog(InternalTraceLevel level, string message)
             writer.WriteLine(TRACE_FMT,
                 DateTime.Now.ToString(TIME_FMT),
                 level == InternalTraceLevel.Verbose ? "Debug" : level.ToString(),
-                System.Threading.Thread.CurrentThread.ManagedThreadId, name, message);
+#if NET20
+                System.Threading.Thread.CurrentThread.ManagedThreadId,
+#elif NETSTANDARD
+                Environment.CurrentManagedThreadId,
+#endif
```

Comment:
Since `Environment.CurrentManagedThreadId` is preferred, can we special-case NET20 as the odd one out and just use `#else`?